<launch>
    <!-- Camera Parameters -->
    <arg name="camera_id" default="0" />
    <arg name="camera_resolution" default="SD" /> <!-- Choose between: SD, HD, FHD -->
    <!-- ROS Topic Parameters -->
    <arg name="image_topic" default="/zed/left_image" />

    <!-- Node -->
    <node name="webcam_cpp" pkg="image" type="webcam_cpp" output="screen">
        <!-- Set Parameters -->
        <param name="camera_id" value="$(arg camera_id)" />
        <param name="camera_resolution" value="$(arg camera_resolution)" />
        <param name="image_topic" value="$(arg image_topic)" />
    </node>
</launch>
